ci/jenkinsfiles/release.groovy

```diff
@@ -0,0 +1,404 @@
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* Contributors:
+*     Abdoul BA <aba@nuxeo.com>
+*/
+
+def appName = 'nuxeo-retention'
+def pipelineLib
+def repositoryUrl = 'https://github.com/nuxeo/nuxeo-retention/'
+
+properties([
+  [
+    $class: 'BuildDiscarderProperty',
+    strategy: [
+      $class: 'LogRotator',
+      daysToKeepStr: '15', numToKeepStr: '10',
+      artifactNumToKeepStr: '5'
+    ]
+  ],
+  [
+    $class: 'GithubProjectProperty', projectUrlStr: repositoryUrl
+  ],
+  disableConcurrentBuilds(),
+])
+
+String currentVersion() {
+  return readMavenPom().getVersion()
+}
+
+String getReleaseVersion(String version) {
+  return version.replace('-SNAPSHOT', '')
+}
+
+pipeline {
+  agent {
+    label 'builder-maven-nuxeo-11'
+  }
+  parameters {
+    string(name: 'rcVersion', description: 'Version to be promoted')
+    string(name: 'reference', description: 'Reference branch to be bumped after releasing')
+    booleanParam(
+      name: 'dryRun', defaultValue: true,
+      description: 'if true all steps will be run without publishing the artifact'
+    )
+  }
+  environment {
+    APP_NAME = "${appName}"
+    BACKEND_FOLDER = "${WORKSPACE}/nuxeo-retention"
+    BRANCH_NAME = GIT_BRANCH.replace('origin/', '')
+    BRANCH_LC = "${BRANCH_NAME.toLowerCase().replace('.', '-')}"
+    CLEANUP_PREVIEW = 'true'
+    CONNECT_PROD_URL = 'https://connect.nuxeo.com/nuxeo'
+    CHART_DIR = 'ci/helm/preview'
+    CURRENT_VERSION = currentVersion()
+    ENABLE_GITHUB_STATUS = 'false'
+    FRONTEND_FOLDER = "${WORKSPACE}/nuxeo-retention-web"
+    JENKINS_HOME = '/root'
+    MAVEN_DEBUG = '-e'
+    MAVEN_OPTS = "${MAVEN_OPTS} -Xms512m -Xmx3072m"
+    NUXEO_VERSION = '2021.0.10'
+    NUXEO_BASE_IMAGE = "docker-private.packages.nuxeo.com/nuxeo/nuxeo:${NUXEO_VERSION}"
+    ORG = 'nuxeo'
+    PREVIEW_NAMESPACE = "retention-${BRANCH_LC}"
+    REFERENCE_BRANCH = '2021'
+    IS_REFERENCE_BRANCH = "${BRANCH_NAME == REFERENCE_BRANCH}"
+    VERSION = getReleaseVersion(CURRENT_VERSION)
+  }
+  stages {
+    stage('Check parameters') {
+      steps {
+        script {
+          if (!params.rcVersion || params.rcVersion == '') {
+            currentBuild.result = 'ABORTED'
+            currentBuild.description = 'Missing required parameter version, aborting the build.'
+            error(currentBuild.description)
+          }
+
+          if (!params.reference || params.reference == '') {
+            currentBuild.result = 'ABORTED'
+            currentBuild.description = 'Missing required parameter reference, aborting the build.'
```

Comment:
currentBuild.description = 'Missing required parameter reference, aborting the build.' ->  currentBuild.description = 'Missing required reference parameter, aborting the build.'